comp.md

```diff
@@ -0,0 +1,64 @@
+# V2
+
+
+context: component repository
+
+- one repo for open source gardener
+- one repo for internal sap repo
+- one repo for alicloud/ns2/on-prem repo
+
+```yaml
+name: gardener
+version: 1.2.3
+
+provider: external | internal
+
+transportContexts:
+- baseUrl: eu.gcr.io/dev-env # origin
+- baseUrl: eu.gcr.io/internal
+- baseUrl: artifactory.wdf.sap.corp
```

Comment:
I'd not expose internal URLs in a public repo. Same for the other ones like github.wdf...

Reply:
the whole file wasn't meant to be committed.
Removed it completely